dll/win32/shell32/dialogs/item_prop.cpp

```diff
@@ -163,8 +161,7 @@ SHELL32_ShowShellExtensionProperties(const CLSID *pClsid, IDataObject *pDO)
 HRESULT
 SHELL_ShowItemIDListProperties(LPCITEMIDLIST pidl)
 {
-    if (!pidl)
-        return E_INVALIDARG;
+    assert(pidl);
```

Comment:
In release builds there is no assert, so if the check is removed then the code will crash in a release build.

Reply:
In this specific case it will actually try to show the properties for the desktop on null pidl. Not what the caller expected but they are not supposed to pass null in the first place.